vnext/Microsoft.ReactNative/ReactPackageBuilder.h

```diff
@@ -11,14 +11,14 @@
 
 namespace winrt::Microsoft::ReactNative {
 
-struct ReactPackageBuilder
-    : winrt::implements<ReactPackageBuilder, IReactPackageBuilder, IReactPackageBuilderExperimental> {
+struct ReactPackageBuilder : winrt::implements<ReactPackageBuilder, IReactPackageBuilder> {
   ReactPackageBuilder(
       std::shared_ptr<NativeModulesProvider> const &modulesProvider,
 #ifndef CORE_ABI
       std::shared_ptr<ViewManagersProvider> const &viewManagersProvider,
 #endif
-      std::shared_ptr<TurboModulesProvider> const &turboModulesProvider) noexcept;
+      std::shared_ptr<TurboModulesProvider> const &turboModulesProvider,
+      bool useTurboModulesByDefault) noexcept;
```

Comment:
>useTurboModulesByDefault [](http://example.com/codeflow?start=11&length=24)

Please make this parameter match the .cpp code. There it is called `isWebDebugging`.